alpha/lib/model/entry.php

```diff
@@ -4030,9 +4030,9 @@ public function getObjectParams($params = null)
 			'entitled_kusers_view' => $this->getEntitledKusersViewArray(),
 			'kuser_id' => $this->getKuserId(),
 			'creator_kuser_id' => $this->getCreatorKuserId(),
-			'name' => $this->getName(),
-			'description' => $this->getDescription(),
-			'tags' => explode(',', $this->getTags()),
+			'name' => explode(',', multiLingualUtils::getElasticFieldValue($this, self::NAME)),
```

Comment:
instead of concat then explode why not just return array from the getElastic.. function?